static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -170,10 +185,20 @@ const canShow = async () => {
 
     const brazeSwitch = config.get('switches.brazeSwitch');
     const apiKey = config.get('page.brazeApiKey');
+    const isBrazeConfigured = brazeSwitch && apiKey;
+    if (!isBrazeConfigured) {
+        return false;
+    }
+
+    const [brazeUuid, hasGivenConsent] = await Promise.all([getBrazeUuid(), hasRequiredConsents()]);
+
+    await maybeWipeUserData(apiKey, brazeUuid);
```

Comment:
This is a super minor point, but technically we don't need to block on the `hasRequiredConsents` step here. We could call `maybeUserData` awaiting just the `brazeUuid` instead of the `Promise.all`. I'm not fussed either way, and code clarity should always have a strong voice in the balance against minor performance tweaks!

Reply:
Ah yep that's a totally valid point. I'd be inclined to leave this as-is and investigate separately. Given that we'll need both the brazeUuid and hasGivenConsent to proceed with showing the banner (after potentially wiping) and there's a timeout around the banner decision, it might make sense to leave those two async operations happening concurrently even if it means blocking `maybeWipeUserData` slightly longer. I'll take a look at benchmarking this to see what difference moving things around makes.